core/utils/ether.h

```diff
@@ -25,6 +25,8 @@ struct[[gnu::packed]] Ethernet {
     // Returns "aa:bb:00:11:22:33" (all in lower case)
     std::string ToString() const;
 
+    void CopyFrom(const Address &o);
```

Comment:
I agree, in most cases.  Here, since I've seen slightly different behavior in various libraries in their use of CopyFrom, I figured clarity is preferable.

However, looking at it more closely, right now it appears the copy constructor and assignment operator are not deleted, so is this method even needed?